Sources/StreamChat/Query/ChannelListQuery.swift

```diff
@@ -47,46 +47,91 @@ extension FilterKey where Scope: AnyChannelListFilterScope {
     static var members: FilterKey<Scope, UserId> { "members" }
 }
 
-/// Non extra-data-specific filer keys for channel list.
+/// Filter values to be used with `.invite` FilterKey.
+public enum InviteFilterValue: String, FilterValue {
+    case pending
+    case accepted
+    case rejected
+}
```

Comment:
Any reason we don't use a struct here? Are we sure this enum won't be changed? Would it be useful for customers to extend this?

Reply:
From backend documentation:
>invite: type string, must be one of these values: (pending, accepted, rejected)